index.html

```diff
@@ -8,31 +8,34 @@
 
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
 
     <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0/angular.js"></script>
     <script src="js/app.js"></script>
-
-
-    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.2/html5shiv.min.js"></script>
-      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-    <![endif]-->
   </head>
-  <body ng-app="GiphyApp">
-    <div class="container">
-      <h1 class="text-center">Welcome to Giphy Search</h1>
 
-      <form class="form-inline">
-        <div class="form-group">
+  <body ng-app="GiphyApp" ng-controller="Ctrl">
+    <div class="container">
+      <h1 class="text-center title">Welcome to Giphy Search</h1>
+      <!-- Search Form -->
+      <form class="form-inline text-center search">
+        <div class="form-group text-center">
           <label for="search">Search</label>
-          <input type="text" placeholder="Search for GIFs" id="search" class="form-control">
+          <input type="text" ng-model="searchTerm" placeholder="Search for GIFs" id="search" class="form-control">
         </div>
-        <button type="submit" class="btn btn-primary">Search</button>
+        <button type="submit" ng-click="search()" class="btn btn-primary">Search</button>
       </form>
-
-      <div id="results"></div>
+      <!-- Search Results -->
+      <div id="results" ng-repeat="result in results track by $index">
+        <div ng-hide="hide" class="well result">
+          <img src="{{ result.images.fixed_height.url }}">
+        </div>
+        <hr>
+      </div>
+      <div ng-hide="patientBear" class="bear">
+        <img src="http://new2.fjcdn.com/comments/Mfw+still+waiting+for+the+gif+to+load+_cf14b8b06d0be4651e41d88734d3920b.jpg">
+      </div>
     </div>
   </body>
+
```

Comment:
Div element to host the infinite scrolling:

    <div class="row" id="results" infinite-scroll="myPagingFunction()" infinite-scroll-distance="3">
        <div class="well col-md-6" ng-repeat="giphy in giphys track by giphy.id">
            <img ng-src="{{giphy.images.fixed_height.url}}" alt="Description" />
            <a ng-href="{{giphy.images.fixed_height.url}}">link</a>
        </div>
    </div>